src/components/ChatUI.tsx

```diff
@@ -205,7 +204,8 @@ const ChatUI: React.FC<ChatUIProps & { embedded?: boolean }> = ({
   >([]);
 
   const portForwardIdRef = useRef<string | null>(null);
-  const [portForwardStatus, setPortForwardStatus] = useState('');
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Using `// eslint-disable-next-line no-unused-vars` is an anti-pattern to fix linting errors unless it's absolutely necessary. Could you either delete the unused variables, or use underscore variables?

<img width="676" height="346" alt="Image" src="https://github.com/user-attachments/assets/03cdaa99-7c7e-4447-b234-375d5ec53071" />